README.md

```diff
@@ -6,20 +6,29 @@ Context-oriented Programming (COP) for JavaScript
 ## Example Code 
 
 ```JS
-import * as cop from "https://lively-kernel.org/lively4/ContextJS/Layers.js"
+import "https://lively-kernel.org/lively4/ContextJS/ContextJS.js"
```

Comment:
An idea we just had to solve this: have the main entry point point to a file that writes the things into global / window namespace instead of letting it point to contextjs.js, because imports without specifying * or { something } only import side effects (which would be putting the stuff into global namespace).